JvmTest fails on MacOS X with Intel processor

Running the "test" target generates the following error on my machine:

[ERROR] Tests run: 20, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 1.691 s <<< FAILURE! - in net.openhft.chronicle.core.JvmTest
[ERROR] net.openhft.chronicle.core.JvmTest.getCpuClass  Time elapsed: 0.288 s  <<< FAILURE!
org.junit.ComparisonFailure: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz expected:<[Apple M1]> but was:<[Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz]>
	at net.openhft.chronicle.core.JvmTest.getCpuClass(JvmTest.java:291)

Root cause is in Boorstrap.java, line 72:

    static boolean isMacArm0() {
        return OS_NAME.equals("Mac OS X");
        // TODO FIX && CpuClass.CPU_MODEL.startsWith("Apple M");
    }

I have noticed that because isMacArm0() erroneously returns true on my machine, there are several errors that also occur in Chronicle-Bytes. I will report these errors seperately.

--- a/src/main/java/net/openhft/chronicle/core/Bootstrap.java
+++ b/src/main/java/net/openhft/chronicle/core/Bootstrap.java
@@ -72,8 +72,7 @@ final class Bootstrap {
     }
 
     static boolean isMacArm0() {
-        return OS_NAME.equals("Mac OS X");
-        // TODO FIX && CpuClass.CPU_MODEL.startsWith("Apple M");
+        return OS_NAME.equals("Mac OS X") && !OS_ARCH.equals("x86_64");
     }
 
     static boolean isAzulZing0() {

